File: Assignments/Assignment 35/C Codes/program35_3.c
// Write a program which accepts a string from user and get difference b/w count of small and 
// count of capital letters

#include<stdio.h>

int Difference(char str[])
{
    int iCountS = 0;
    int iCountC = 0;

    while(*str != '\0')
    {
        if(*str >= 'A' && *str <= 'Z')
        {
            iCountC++;
        }
        else if(*str >= 'a' && *str <= 'z')
        {
            iCountS++;
        }
        str++;
    }

    return (iCountS-iCountC);
}

int main()
{
    char Arr[50] = {'\0'};
    int iRet = 0;

    printf("Enter String : \n");
    scanf("%[^'\n']s", Arr);

    iRet = Difference(Arr);

    printf("%d\n", iRet);

    return 0;
}

File: Assignments/Assignment 35/Java Codes/program35_3.java
// Write a program which accepts a string from user and get difference b/w count of small and 
// count of capital letters

import java.util.*;

class Test 
{
    int Difference(String str)
    {
        char Arr[] = str.toCharArray();  // Converted String to Array of character
        int iCnt  = 0; 
        int iCountSmall = 0;
        int iCountCapital = 0;

        for(iCnt = 0; iCnt < Arr.length; iCnt++)        // Important
        {
            if(Arr[iCnt] >= 'a' && Arr[iCnt] <= 'z')
            {
                iCountSmall++;
            }
            else if(Arr[iCnt] >= 'A' && Arr[iCnt] <= 'Z')
            {
                iCountCapital++;
            }
        }

        return (iCountSmall-iCountCapital);
    }
}

class program35_3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iRet  = 0;

        System.out.println("Enter String : ");
        String Arr = sobj.nextLine();

        Test tobj = new Test();

        iRet = tobj.Difference(Arr);

        System.out.println("Frequency is : " + iRet);
    }
}